test/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+networks:
+  public:
+    name: scylla_rust_driver_public
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 172.42.0.0/16
+services:
+  scylla1:
+    image: scylladb/scylla
+    ports:
+      - 127.0.0.1:9042:9042
+      - 127.0.0.1:9142:9142
+      - 127.0.0.1:19042:19042
+      - 127.0.0.1:19142:19142
```

Comment:
Are those published ports used? The nodes are using `172.x.x.x` addresses as `rpc-address`, so the driver will attempt to connect to that address directly rather than on localhost. I assume that everything is running on Linux, so the `172.x.x.x` addresses are reachable.

Perhaps we could get rid of the published ports to reduce confusion, and set SCYLLA_URI appropriately when running tests.